_posts/en/newsletters/2020-09-23-newsletter.md

```diff
@@ -0,0 +1,291 @@
+    <-> relay   net mping   ping send recv  txn  blk uptime  asmap    id version
+     in  full onion                 1    1                0        10206 70015/Satoshi:0.20.1/
+     in  full  ipv6   246    246    1    9    0           1  16509 10202 70015/Satoshi:0.19.1/
+     in block onion 37686 425955   42   42               25        10143 70015/Satoshi:0.20.1/
+    [...]
+    out  full  ipv4    94    198    3    1    0  229    956  12998  7809 70015/Satoshi:0.19.0.1/
+    out block  ipv6   107    269   64   64              949   5577  7835 70015/Satoshi:0.16.0/
+    out  full onion   440   1180    4    4    0         257         9574 70015/Satoshi:0.18.1/
+                       ms     ms  sec  sec  min  min    min{% comment %}skip-duplicate-words-test{% endcomment %}
+
+            ipv4    ipv6   onion   total  block-relay
+    in         0      17       8      25       2
+    out        8       2       8      18       2
+    total      8      19      16      43       4
+
+    Local addresses
+    [redacted]                                port  8333     score   6401
+    [redacted].onion                          port  8333     score   1085
+    ```
+
+- [Bitcoin Core #15454][] no longer creates a new wallet when the
+  program is started for the first time.  Instead, the user is prompted
+  to either load an existing wallet or create a new named wallet.
+  Wallets created by default in earlier versions of the software will
+  still be loaded by default, and newly created wallets can still be
+  added to the load-on-start list (see [Newsletter #111][news111
+  load_on_startup]).  By removing the default wallet creation, users
+  gain more exposure to the options for customizing their wallet type,
+  such as watch-only wallets or wallets that generate multisig
+  addresses.
+
+- [Bitcoin Core #19940][] updates the `testmempoolaccept` RPC to also
+  return the transaction's vsize and total transaction fee if the
+  transaction can be added to the mempool.  In particular, the fee is
+  information that some users need, which is guaranteed to be available
+  for a mempool transaction, and which cannot be trustlessly obtained
+  before broadcasting a non-wallet transaction unless several other RPCs
+  are run in sequence.
+
+- [LND #4440][] records the number of times the local node sees a peer
+  going offline and then coming back online, known as flapping.  The
+  node will limit the number of event records it'll store about peers
+  who frequently flap to avoid filling its database with too much noise.
+  The `listchannels` RPC is also updated to display each peer's flap
+  rate.
+
+- [LND #4567][] adds a new `--maxchansize` configuration parameter that
+  allows setting the maximum amount of money that can be contained in a
+  new channel.  Now that LND supports [large channels][topic large
+  channels] (see [Newsletter #107][news107 lnd wumbo]), this setting
+  allows users to set a limit on the maximum amount of money they'll
```

Comment:
I suggest softening this to "they could potentially lose". After all, something could go wrong and the user not lose their money!